kolibri/plugins/coach/assets/src/views/plan/CoachExamsPage/index.vue

```diff
@@ -408,6 +457,17 @@
         nextRoute.name = nextRouteName;
         this.$router.push(nextRoute);
       },
+      fetchResources() {
+        ChannelResource.fetchCollection({
+          getParams: {
+            contains_exercise: true,
+            available: true,
+            contains_quiz: true,
```

Comment:
[blocking] `contains_quiz` is not the most intuitive, because this isn't actually about for being able to create a quiz, but rather, whether or not a channel contains practice quizzes. Including this as `true` therefore is more restriction than you want. 

`contains_exercise: true` is definitely always required for a channel to be able to be used to create a quiz, but `contains_quiz` is not required. 

To see more specifically what I mean, you can use the Kolibri QA channel. Importing a few exercises, but no practice quizzes means that this fetch will return an empty array.

https://github.com/user-attachments/assets/15396efc-8336-4b38-baa4-44db4cd39334